packages/systemd/systemd.spec

```diff
@@ -61,6 +61,15 @@ Patch9012: 9012-core-mount-increase-mount-rate-limit-burst-to-25.patch
 # lease in EC2.
 Patch9013: 9013-sd-dhcp-lease-parse-multiple-domains-in-option-15.patch
 
+# Backport of upstream patches that make the netlink default timeout
+# configurable.  Bottlerocket carries this patch and configures the timeout in
+# an effort to avoid a situation where a network link becomes unusable if the
+# system is under load and doesn't process the RTM_NEWROUTE acknowledgement
+# within the default timeout of 25 seconds.
+# Reference issue: github.com/systemd/systemd/issues/25441
+Patch9014: 9014-sd-netlink-make-calc_elapse-return-USEC_INFINITY-whe.patch
+Patch9015: 9015-sd-netlink-make-the-default-timeout-configurable-by-.patch
```

Comment:
These should be numbered differently - say 1001 and 1002 - since we expect to be able to drop them after a future rebase.

Local patches like the 9000 series should always be applied after any upstream patches since we want to rebase local changes immediately and not have to adjust backported patches to work around our changes.